packages/Interface/src/DTK_C_API.h

```diff
@@ -94,16 +98,85 @@ extern DTK_UserApplicationHandle DTK_create( DTK_ExecutionSpace space );
  *  \return true if the given user application handle is valid;  false
  *  otherwise.
  */
-extern bool DTK_is_valid( DTK_UserApplicationHandle handle );
+extern bool DTK_isValidUserApplication( DTK_UserApplicationHandle handle );
 
-/** \brief Destroy a DTK handle.
+/** \brief Destroy a DTK handle to a user application.
  *
  *  \param[in,out] handle User application handle.
  */
-extern void DTK_destroy( DTK_UserApplicationHandle handle );
+extern void DTK_destroyUserApplication( DTK_UserApplicationHandle handle );
+
+/**@}*/
+
+
+/**
+ * \defgroup c_interface_to_map Interface to maps.
+ * @{
+ */
+
+/** \brief DTK map handle.
+ *
+ *  Must be created using DTK_createMap() to be a valid handle.
+ *
+ *  The handle essentially hides C++ implementation details from the user.
+ *
+ *  <!--
+ *  Use incomplete types to differentiate between handles.
+ *  We never define the incomplete structs.
+ *  -->
+ */
+typedef struct _DTK_MapHandle *DTK_MapHandle;
```

Comment:
Do we prefer "Map" over "Operator"?

Reply:
I think "Map" is a bit more intuitive for those without an extensive linear algebra background. It simply "maps" the data from one code to another. Although I guess "TransferOperator" is also equally intuitive as long as the word "transfer" is included.